src/components/Results/use/DownloadForm.ts

```diff
@@ -0,0 +1,130 @@
+import { ResultsPerQuestion, isArrayQuestion } from './statistics'
+import { getTimeLimit } from '/@/components/UI/use/useOptions'
+
+type FormInfo = {
+  header: SpreadForm
+  rows: SpreadForm[]
+}
+type SpreadForm = {
+  readonly traq_id: string
+  readonly response_id?: number
+  readonly submitted_at: string
+  results: string[]
+}
+
+const generateHeader = (resultsPerQuestion: ResultsPerQuestion) => {
+  const spreadHeader: SpreadForm = {
+    traq_id: 'traQID',
+    submitted_at: '回答日時',
+    results: []
```

Comment:
```suggestion
    results: resultsPerQuestion.questions.map(v => v.question.body)
```
これぐらいならワンライナーで書いてほしいです